string command
---
float32 value
int32 res
string error